packages/ra-supabase-core/README.md

```diff
@@ -0,0 +1,166 @@
+# ra-supabase-core
+
+This package provide a dataProvider, an authProvider and hooks to integrate [Supabase](https://supabase.io/) with [react-admin](https://marmelab.com/react-admin).
+
+## Usage
+
+```jsx
+// in supabase.js
+import { createClient } from '@supabase/supabase-js';
+
+export const supabase = createClient(
+    SUPABASE_URL,
+    SUPABASE_ANON_KEY
+);
+
+// in dataProvider.js
+import { supabaseDataProvider } from 'ra-supabase';
```

Comment:
you import from 'ra-supabase' in this example, yet the current package is called 'ra-supabase-core'. I'm not sure I understand the difference

Reply:
`ra-supabase` is a distribution like react-admin. We provide UI components through the `ra-supabase-ui-materialui` package